manifests/init.pp

```diff
@@ -252,14 +252,22 @@
   $cluster_node_timeout                                          = $redis::params::cluster_node_timeout,
   Integer[0] $cluster_slave_validity_factor                      = $redis::params::cluster_slave_validity_factor,
   Boolean $cluster_require_full_coverage                         = $redis::params::cluster_require_full_coverage,
-  Integer[0] $cluster_migration_barrier                          = $redis::params::cluster_migration_barrier
+  Integer[0] $cluster_migration_barrier                          = $redis::params::cluster_migration_barrier,
+  ## Start hiera lookup
```

Comment:
I think this comment can go. You might be using hiera, but if someone wanted to use `instances` directly in their declaration of the class, that would work equally well.

Reply:
Done.

Reply:
Looking good.  Just need to add a puppet strings style comment up [here](https://github.com/voxpupuli/puppet-redis/pull/330/files#diff-60ae41fd0a31977447947f59940ee9a4L155) somewhere, and I think we're good to merge.  (You could regenerate the Reference.md, but not essential as I'm sure someone else will before the next release anyway)